yb-voyager/cmd/importDataState.go

```diff
@@ -394,7 +394,7 @@ func (bw *BatchWriter) WriteRecord(record string) error {
 		return nil
 	}
 	var err error
-	if bw.flagFirstRecordWritten {
+	if bw.flagFirstRecordWritten && !strings.HasSuffix(record, "\n") { //TODO: check if this is better way to check if \n is already added or not
```

Comment:
I assume this is because the csvWriter is appending a \n at the end. 
How about preemptively removing that in ConvertRow with strings.TrimSuffix ? it